### PR TITLE
Feature/datacite publisher

### PR DESCRIPTION
**Pull request by southeo**
Will forever regret calling the "handle-manager" api that...

In acceptance (as in production) we want to have 2 deployments of the PID api: handle and doi. media and specimen processors will request pids from the doi deployment. the doi deployment of the api will send messages to the datacite publisher, which will create test dois with datacite.

I wasn't sure if we wanted to change existing deployments now or after we do a release to acceptance. for now i've kept the specimen and media processors the same, but they should eventually point to the doi version of the api - there's one env var to change there.

### CODE REVIEW

File: acceptance/datacite-publisher/datacite-publisher.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        language: java
+      annotations:
+        co.elastic.logs/multiline.pattern: '^[[:space:]]+(at|\.{3})[[:space:]]+\b|^Caused by:'
+        co.elastic.logs/multiline.negate: "false"
+        co.elastic.logs/multiline.match: after
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - name: datacite-publisher
+          image: public.ecr.aws/dissco/datacite-publisher@sha256:d669eefec83bec9f2a49a71dfb5d551cd86f1a57fd213d594d3207a39740d26b
```
Comment (samleeflang) [inline acceptance/datacite-publisher/datacite-publisher.yaml:25]: Why was this sha so long again and can we change it?

File: acceptance/datacite-publisher/datacite-publisher.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          imagePullPolicy: Always
+          ports:
```
Comment (samleeflang) [inline acceptance/datacite-publisher/datacite-publisher.yaml:27]: We need both ports?

Comment (samleeflang) [reply acceptance/datacite-publisher/datacite-publisher.yaml:27]: Looking at the files the 8080 might be enough

File: acceptance/handle-manager/handle-manager-deployment-doi.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        co.elastic.logs/multiline.pattern: '^[[:space:]]+(at|\.{3})[[:space:]]+\b|^Caused by:'
+        co.elastic.logs/multiline.negate: "false"
+        co.elastic.logs/multiline.match: after
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - name: dissco-handle-manager
+          image: public.ecr.aws/dissco/handle-manager:sha-6d5eec8
+          imagePullPolicy: Always
+          ports:
```
Comment (samleeflang) [inline acceptance/handle-manager/handle-manager-deployment-doi.yaml:27]: I don't think we need the 80 looking at the route and the health points

**Review by samleeflang — COMMENTED** [review]
:ocean:

File: acceptance/handle-manager/handle-manager-deployment-doi.yaml (hunk 3 above)
Comment (southeo) [reply acceptance/handle-manager/handle-manager-deployment-doi.yaml:27]: should i also remove the health points for the datacite publisher?

Comment (southeo) [reply acceptance/handle-manager/handle-manager-deployment-doi.yaml:27]: do you mean remove he port or should i also remove the health checks?

**Review by samleeflang — APPROVED** [review]
Nice, happy the sha got fixed!

**Pull request merged**